scripts/hooks/pre-commit

```diff
@@ -4,7 +4,13 @@
 
 STATUS=0
 
-./run_formatter.sh
+if [ -f ./run_formatter.sh ]; then
```

Comment:
but the hook may run on a version which isn't up to date because the hook doesn't change when you checkout a different branch